sw/host/tests/rom/e2e_bootup_bad_rom_ext_signature/src/main.rs

```diff
@@ -37,10 +38,15 @@ fn corrupt_rom_ext(rom_ext_path: &PathBuf) -> Result<NamedTempFile> {
         Err(error) => panic!("Problem reading the file: {:?}", error),
     };
 
-    // Flip a bit within the signature field of the ROM_EXT manifest
-    buffer[80] = &buffer[80] ^ 0b10;
+    // Flip random bits at a random index.
+    let mut rng = thread_rng();
+    let index = rng.gen_range(0..buffer.len());
+    let mask = rng.gen_range(1..std::u8::MAX);
+    // Logging to allow reproducibility.
+    println!("Masking the byte {} with {}", index, mask);
+    buffer[index] = &buffer[index] ^ mask;
```

Comment:
The Bazel clippy rule wants this simplified to `buffer[index] ^= mask;` (unnecessary reference, and re-implementing the assignment operator `^=`)

Reply:
Neat!